docs/concepts/manifest.md

```diff
@@ -1,325 +0,0 @@
-
-The Flox catalog contains a hierarchy of packages where some are at the top
-level,
-and others are contained under package sets.
-The `pkg-path` is the location of the package within the catalog,
-including the package sets that it may be nested under.
-For instance,
-`pip` is nested under `python3Packages`,
-so its `pkg-path` is `python3Packages.pip`.
-The `pkg-path` is also what's displayed in search results,
-so you should be able to copy and paste a search result into your manifest
-or use it in a [`flox install`](../reference/command-reference/flox-install.md)
-command.
-
-#### Specifying package versions
-
-The manifest can **install packages by semantic version** (semver).
-The package will only be installed if it has a semantic version that falls
-within the filter provided via the `version` key.
-You use the `@` character to join the package name with the version when using
-the CLI.
-
-Let's try installing `curl` with at least version 8.
-
-```console
-$ # quoting is necessary for search terms that contain '>' or '<'
-$ flox install 'curl@>=8'
-```
-
-Notice that the manifest now contains a second line with the semantic version
-filter.
-
-```toml
-[install]
-curl.pkg-path = "curl"
-curl.version = ">=8"
-```
-
-You may also request a specific version
-Here is an example of installing `curl` version 8.1.1:
-
-``` console
-$ flox install curl@8.1.1
-```
-
-Notice that the version line now starts with `=`.
-This is how you tell Flox to install exact versions or versions that don't
-adhere to semantic versioning.
-
-``` toml
-[install]
-curl.pkg-path = "curl"
-curl.version = "=8.1.1"
-```
-
-#### Installing packages to package groups
-
-Flox will try to install packages that have been known to work together by default.
-This allows Flox to ensure maximum compatibility and has the benefit of keeping the environment as small as possible.
-However, sometimes you may need software that varies in age: For example, `packageA` you want to be from last week while `packageB` you need to be a specific older version. In these cases, you may see Flox error saying the constraints are too tight. To resolve this, you can specify a separate collection of packages using their `pkg-group` attribute.
-
-```toml
-[install]
-packageA.pkg-path = "packageA"
-
-packageB.pkg-path = "packageB"
-packageB.version = "some.old.version"
-packageB.pkg-group = "backend" # (1)!
-```
-
-1. "backend" is an arbitrary name. Try naming your pkg-group matching the logical grouping.
-
-`pkg-group` is also useful for ensuring maximum compatibility between packages.
-In this example say you're developing a new machine learning library that
-depends on the XGBoost ML library.
-XGBoost links against the popular Boost C++ collection of libraries,
-and if we want to write our own C++ code that uses Boost,
-we'll want to link against the same version of Boost that XGBoost is using.
-We can ensure this happens by placing Boost and XGBoost in the same `pkg-group`.
```

Comment:
nonblocking question: would it be good to add any of this to the man page, or would examples feel out of place?